qdrant-landing/content/documentation/quickstart/quickstart-langchain.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Start with LangChain
+weight: 1
+---
+
+# Chat With Your Documents 
+
+This is a very short tutorial that shows you how to retrieve information from your documents using Qdrant, OpenAI and LangChain.
+You will load a text file into Qdrant and ask a few basic questions. Qdrant will retrieve semantically appropriate answers. This process is fun and we recommend you try this with all sorts of books, journal articles and other content you wish to break down efficiently.  
+
+## Prerequisites
+- You can use any text file. For your convenience, [download our short story]().
+- To create embeddings from this text file, [you will need an OpenAI API key](https://help.openai.com/en/articles/4936850-where-do-i-find-my-secret-api-key). 
+- To retrieve answers with Qdrant, [create a Qdrant Cloud Cluster and an API key](/documentation/cloud/create-cluster/).
+
+## Initialize libraries
+
+In this example, we are using LangChain, Qdrant's Python Client, OpenAI and its tokenizer.
+
+```bash
+pip install langchain qdrant_client openai tiktoken
+```
+
+```python
+from langchain.vectorstores import Qdrant
+from langchain.embeddings.openai import OpenAIEmbeddings
+import qdrant_client
+import os
+```
```

Comment:
If those code snippets are put next to each other, they are rendered as tabs.
![image](https://github.com/qdrant/landing_page/assets/2649301/a8f68d5b-b921-46ae-a73f-eba9ca0fe73a)
It feels like we have to choose one of the options, but in reality, we need to make both steps.